docs/stability.rst

```diff
@@ -0,0 +1,197 @@
+            <th class="head">
+                Subpackage
+            </th>
+            <th class="head">
+                &nbsp;
+            </th>
+            <th class="head">
+                Comments
+            </th>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.atomic
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                This package is being actively developed and expanded,
+                and there may be backward incompatible changes to the API.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.classes
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                The plan for PlasmaPy's base classes is being planned in
+                PLEP 7, which is in the process of being written.  The
+                existing functionality is unstable.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.constants
+            </td>
+            <td align='center'>
+                <span class="stable"></span>
+            </td>
+            <td>
+                We do not anticipate that there will be any major backward
+                incompatible changes within the
+                <tt class="docutils literal"><span class="pre">constants</span></tt>
+                subpackage.  However, the values for constants may be updated
+                when improved values become available.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.diagnostics
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                This subpackage is in the early stages of development.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.mathematics
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                The
+                <tt class="docutils literal"><span class="pre">mathematics</span></tt>
+                subpackage is in the early stages of development.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.parameters
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                The
+                <tt class="docutils literal"><span class="pre">parameters</span></tt>
+                subpackage may undergo significant reorganization with potentially
+                major changes to the API.  We anticipate that there will be major
+                additions to this package.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.parameters.transport
```

Comment:
Same here, I'm pretty sure transport is a submodule of physics and not parameters :smile: